_data/finance.yml

```diff
@@ -46,6 +46,12 @@ websites:
       tfa: Yes
       sms: Yes
       phone: Yes
+      
+    - name: DEGIRO
+      url: https://www.degiro.nl
+      img: degiro.png
+      tfa: Yes
+      software: Yes
 
```

Comment:
Can you provide some form of documentation on how to enable 2FA on DEGIRO?
If not, could you please attach a screenshot of the login process or 2FA setup process? (With personal information blurred out)